app/src/main/java/plantfueled/puppysitter/Pet.java

```diff
@@ -11,6 +13,10 @@
     private float hungerLevel = 100;
     private float lonelyLevel = 100;
     private Date lastPetUpdate;
+    private PetNotification petNotification;
```

Comment:
I think its a bit awkward that the Pet has a PetNotification object and the PetNotification has a Pet object.  Maybe changing some of the methods in the PetNotification class so that it does not require storing a reference to a Pet.  I think the logic for knowing when the Pet has changed status can be implemented in the Pet class instead of PetNotification so then Pet can be the one to call petNotification.lonelyChange(...) and petNotification.hungerChange(...)